sr_hand/launch/gazebo/gazebo_hand.launch

```diff
@@ -5,7 +5,7 @@
   <arg name="headless" default="false" />
   <arg name="debug" default="false" />
   <arg name="verbose" default="false" />
-  <arg name="prefix" default="rh_" />
+  <arg name="hand_id" default="rh" />
```

Comment:
https://github.com/shadow-robot/sr-ros-interface-ethercat/blob/indigo-devel/sr_edc_launch/sr_edc.launch#L25

This one is still missing. `hand_id` is not really unique / easy to find so won't be used by the parser.